src/js/pouchPersisted.js

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe, PouchDB */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Base grade for persistence of model components to Pouch
+    // Implementing grades should ensure that what's integral to
+    // reconstructing the particular instance is stored in the
+    // model
+    fluid.defaults("floe.dashboard.pouchPersisted", {
+        gradeNames: ["floe.dashboard.eventInTimeAware"],
+        model: {
+            // Stores the typename to assist in reconstruction
+            // of stored model components in another context
+            "persistenceInformation": {
+                "typeName": "{that}.typeName"
```

Comment:
Point taken - the typeName persistence is actually an artifact of the rapidly developed prototype (which actually does need to persist components to PouchDB for later reconstruction) that should have been stripped out for this PR. :)